From 53b9f97fb7faed043134f2ab738347b20379d32f Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 6 Jun 2017 07:39:28 -0700
Subject: [PATCH 5/6] Undefine PAGE_SIZE

musl defines PAGE_SIZE and build complains
of multiple definitions

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/lib/src/fwts_acpi_tables.c | 1 +
 1 file changed, 1 insertion(+)

--- a/src/lib/src/fwts_acpi_tables.c
+++ b/src/lib/src/fwts_acpi_tables.c
@@ -40,6 +40,7 @@
 #define BIOS_START	(0x000e0000)		/* Start of BIOS memory */
 #define BIOS_END  	(0x000fffff)		/* End of BIOS memory */
 #define BIOS_LENGTH	(BIOS_END - BIOS_START)	/* Length of BIOS memory */
+#undef PAGE_SIZE
 #define PAGE_SIZE	(4096)
 
 static fwts_acpi_table_info	tables[ACPI_MAX_TABLES];
